Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -13,7 +13,119 @@
       <commands:NewFileCommand x:Key="NewFile" />
       <commands:GenericCommand x:Key="GenericCommand" />
       <converters:StringFormatConverter x:Key="StringFormatConverter" />
+
+      <StaticResource x:Key="MenuFlyoutItemKeyboardAcceleratorTextForeground" ResourceKey="SystemControlForegroundBaseMediumBrush" />
+      <StaticResource x:Key="MenuFlyoutItemKeyboardAcceleratorTextForegroundPointerOver" ResourceKey="SystemControlHighlightAltBaseMediumBrush" />
+      <StaticResource x:Key="MenuFlyoutItemKeyboardAcceleratorTextForegroundPressed" ResourceKey="SystemControlHighlightAltBaseMediumBrush" />
+      <StaticResource x:Key="MenuFlyoutItemKeyboardAcceleratorTextForegroundDisabled" ResourceKey="SystemControlDisabledBaseMediumLowBrush" />
     </ResourceDictionary>
+
+    <Style TargetType="MenuFlyoutItem">
```

Comment:
I literally ripped this from the MenuFlyoutItem template in the generic.xaml for 10.0.17110.0, as it appears to be what Ani based her design on. I didn't write it, but it made it pretty easy to implement the design.

Reply:
So this is a new style that shipped in RS4?  @nmetulev do we need to get buy-in to ship that as part of the toolkit?  If it works down-level to RS2 still, then it could be useful to add.

Reply:
Hmm, great question. Let's not include it for now and we can investigate this once we merge this PR.

Reply:
Sounds good